trp_project_scrum/security/ir_rules_collaborators_followers.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
+        <record id="project_rule_foll" model="ir.rule">
+            <field name="name">Show only collaborator follower Project</field>
+            <field name="model_id" ref="project.model_project_project"/>
+            <field name="domain_force">[
+                ('message_follower_ids', 'in', [user.partner_id.id]),
+                ('use_scrum', '=', True)
+                ]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
```

Comment:
it's not so interesting to set this one: https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/ir/ir_rule.py#L78, https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/ir/ir_rule.py#L61